doi/src/main/java/ca/nrc/cadc/doi/GetAction.java

```diff
@@ -127,7 +126,7 @@ private Title getTitle(Resource resource) {
         Title title = null;
         List<Title> titles = resource.getTitles();
         for (Title t : titles) {
-            if (t.titleType == null) {
+            if (StringUtil.hasText(t.getValue())) {
```

Comment:
So `titleType` is a but.

Reply:
titleType is optional, but the title value went from empty not allowed in the old schema, to empty allowed in the new schema. We don't want empty titles, hence the check. Don't know why it was checking for an optional attribute previously, doesn't make sense.